src/db.cpp

```diff
@@ -39,7 +39,7 @@ void CDBEnv::EnvShutdown()
     if (ret != 0)
         printf("EnvShutdown exception: %s (%d)\n", DbEnv::strerror(ret), ret);
     if (!fMockDb)
-        DbEnv(0).remove(strPath.c_str(), 0);
+        DbEnv((u_int32_t)0).remove(strPath.c_str(),0);
```

Comment:
Agree on the space, and drop the stray underscore so we get to a standard `uint32_t` :)